sound/soc/sof/ipc4-topology.h

```diff
@@ -349,6 +349,23 @@ struct sof_ipc4_src {
 	struct sof_ipc4_msg msg;
 };
 
+/**
```

Comment:
commit message: remove references to sink/source pins and only use input/output for consistency